src/client/components/ConsentsSubHeader.stories.tsx

```diff
@@ -53,23 +59,17 @@ export const Newsletters = () => (
 		autoRow={autoRow}
 		title="My Title"
 		current={CONSENTS_PAGES.NEWSLETTERS}
+		isInAbSimplifyRegFlowTest={false}
 	/>
 );
 Newsletters.storyName = 'With Newsletters as current';
-export const OurContent = () => (
-	<ConsentsSubHeader
-		autoRow={autoRow}
-		title="My Title"
-		current={CONSENTS_PAGES.OUR_CONTENT}
-	/>
-);
-OurContent.storyName = 'With Our content as current';
 
 export const YourData = () => (
 	<ConsentsSubHeader
 		autoRow={autoRow}
 		title="My Title"
 		current={CONSENTS_PAGES.YOUR_DATA}
+		isInAbSimplifyRegFlowTest={false}
 	/>
 );
 YourData.storyName = 'With Your Data as current';
```

Comment:
And I suggest adding a story for when it's set to `true`

Reply:
Yeah I have one of those in the main branch